Cargo.toml

```diff
@@ -1,22 +1,30 @@
 [package]
+authors = ["Vincent Prouillet <vincent@wearewizards.io>"]
 name = "tera"
 version = "0.1.0"
-authors = ["Vincent Prouillet <vincent@wearewizards.io>"]
 
 [dependencies]
-walkdir = "0.1"
+log = "^0.3"
+glob = "0.2.11"
```

Comment:
if you use "0.2" in cargo.toml, it will update if "0.2.12" is released automatically which is a bit nicer